addon/components/tether-popover-on-element.js

```diff
@@ -4,53 +4,96 @@ import layout from 'ember-tooltips/templates/components/tether-popover';
 
```

Comment:
This file ([full version here](https://github.com/sir-dunxalot/ember-tooltips/blob/93505b486c93a3cde1095fc2084aee3fdee7b21d/addon/components/tether-popover-on-element.js)) is a good example of what code conventions we're actually enforcing. The conventions and rules are too numerous to list.